Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -281,6 +305,60 @@ public IEnumerator TestRadialSetPrefab()
             Assert.IsFalse(firstRadialButton.GetComponent<Interactable>().IsToggled);
             Assert.IsTrue(secondRadialButton.GetComponent<Interactable>().IsToggled);
             Assert.IsFalse(thirdRadialButton.GetComponent<Interactable>().IsToggled);
+
+            //Cleanup
+            GameObject.Destroy(radialSet);
+        }
+
+        /// <summary>
+        /// Assemble an Interactable GameObject and test various SelectionModes
+        /// </summary>
+        [UnityTest]
+        public IEnumerator TestDimensions()
+        {
+            // Load interactable
+            Interactable interactable;
+            Transform translateTargetObject;
+
+            AssembleInteractableButton(
+                out interactable,
+                out translateTargetObject);
+
+            // Test Button type
+            interactable.Dimensions = 1;
```

Comment:
I don’t understand. There is no way to se selection mode type. It is set via dimensions. 

I guess we could add one...and the. For multi dimension just default to 3?

Reply:
I meant adding test case for setting ButtonMode but now I see there's no setter for that. Perhaps we should add one?

Reply:
Going to leave this out for now since it is covered by Dimensions which creates buttonmode. Add note to #6101 to resolve this setting in inspector and code